wp-content/themes/nuclearnetwork/sass/modules/_print.scss

```diff
@@ -27,7 +27,7 @@
 
             &:after {
 
-            	content: 'Printed from ChinaPower Project, ©2017 by the Center for Strategic and International Studies. All rights reserved.';
+            	content: 'Printed from Next Generation Nuclear Network, ©2017 by the Center for Strategic and International Studies. All rights reserved.';
```

Comment:
Good catch on changing the site name (oops), but the date didn't get updated.

Reply:
Haha, agh! Fixed this.